Rover/Rover.cpp

```diff
@@ -130,12 +130,50 @@ Rover::Rover(void) :
     channel_steer(nullptr),
     channel_throttle(nullptr),
     channel_lateral(nullptr),
+    channel_roll(nullptr),
+    channel_pitch(nullptr),
+    channel_gait_select(nullptr), // for walking robot gait select
+    channel_walkingbot_mode(nullptr), // for walking robot contol mode
     logger{g.log_bitmask},
     modes(&g.mode1),
     control_mode(&mode_initializing)
 {
 }
 
+// get control outputs ( for use in scripting)
+bool Rover::get_control_outputs(uint8_t selector, float &control_value)
```

Comment:
I think this is a good way to do this actually.  Let's just change the "selector" to be an enum.

Reply:
enum with a switch !

Reply:
`enum class` specifically :)